project: add DatabaseStreamer.

mfocshaner opened the pull request.

Commit:
EventModel: update to fit database.

Commit:
UserModel: initial commit.

Commit:
ViewSingleEvent: update EventModel calls.

Commit:
DatabaseStreamer: initial commit.

Commit:
CreateEventActivity: add writing to database.

idosavion commented on app/src/main/java/com/huji/foodtricks/buddies/EventModel.java:
```diff
@@ -57,29 +58,31 @@ public Date getTime() {
         return attendanceProvider.getNonResponsiveCount();
     }
 
+    public boolean isUserOrganizer(String userID) {
```
A function that returns the organizer's ID might be more useful (for sending notification for example)

mfocshaner replied:
will add + merge

idosavion reviewed (dismissed):
Looks great!
we can add the function I suggested when needed, add it or merge it this way if you want :)

Commit:
DatabaseStreamer: add null checks.

Commit:
EventModel: add organizerId getter.

Commit:
UserModel: add array initialization.

If a usermodel is saved to database without any events, when it is read, the \c EventIDs array is null,
therefore before adding an event, it should be re-initialized.
